wiki/Ranking_criteria/osu!catch/en.md

```diff
@@ -183,10 +181,10 @@ Difficulty-specific rules and guidelines do only apply to the difficulty level t
 #### Guidelines
 
 - **[Hyperdashes](/wiki/Gameplay/Hyperdash) should only be used on [drops](/wiki/Gameplay/Hit_object/Juice_stream#drop) and/or slider repetitions when the slider path is simple and easy-to-follow.** This is to prevent chaotic or unreasonably difficult slider movement, as transitioning into and out of complex slider shapes with hyperdash is usually uncomfortable to play, and a major penalty to accuracy for little benefit.
-- **1/8 and higher [hyperdashes](/wiki/Gameplay/Hyperdash) should not be used between consecutive object pairs.**
+- **1/8 and faster [hyperdashes](/wiki/Gameplay/Hyperdash) should not be used between consecutive object pairs.**
 - **[Edge dashes](/wiki/Gameplay/Edge_dash) may be used with caution for a maximum of three consecutive objects, and should not be used after [hyperdashes](/wiki/Gameplay/Hyperdash).**
 - **[Combos](/wiki/Beatmapping/Combo) should not exceed 16 objects including slider tails and repeats.** [Spinners](/wiki/Gameplay/Hit_object/Spinner) are an exception.
-- **Note density should follow a mostly 1/2 + 1/4 and/or 1/3 + 1/6 pattern.** 1/8 patterns and higher should be used sparingly.
+- **Note density should follow a mostly 1/2 + 1/4 and/or 1/3 + 1/6 pattern.** 1/8 and faster patterns should be used sparingly.
```

Comment:
also think this is worse. it's not necessarily faster if there's caps, for instance. revert.

Reply:
this is essentially what prompted the change because currently this language is really inconsistent across RC -- a lot of places do use the wording I suggested already. it also bothered me that "higher" can mean two opposite things in incredibly similar contexts, when talking about beat snaps / rhythm density / etc. (see OP)

but more importantly, I am realising now that I interpreted this in a different way than you did. I thought that by "1/8 patterns or higher", it was talking about patterns that have consecutive notes with 1/8 gaps or smaller. but you seem to have read it as talking about any notes snapped to 1/8 or more precise snaps, which indeed may not support my suggested wording. maybe the wording of this whole line should be questioned... I'll see if I can talk to someone who would know for sure what the intention was

Reply:
Could be "1/8 and beyond" i guess?

Reply:
"denser/sparser rhythms" would also work